### PR TITLE
Da databaseinteraction

### PR DESCRIPTION
**Pull request by danirenae**
# Description

# Related Ticket(s)
This is related to ticket #17 

## Testing

[x] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
[x] I certify that all existing tests pass

## Deploy Notes
Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.

```sh
git pull --prune
git checkout <feature_branch>
```

## Steps to Test
Outline the steps to test the code in this pull request.

1. run python test_payment.py in your terminal to test the payment tests and database interactions. Database table is able to get and set.

### CODE REVIEW

**Comment by pinkston4** [comment]
👍

File: db/payment_db_interactor.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import sqlite3
+
+
+class PaymentDatabaseInteractor():
+
```
Comment (samphillips1879) [inline db/payment_db_interactor.py]: Needs some docstrings 😢

**Review by samphillips1879 — COMMENTED** [review]
Tests pass on my machine 👍 everything looks good, except payment_db_interactor.py needs some docstrings.

**Review by samphillips1879 — COMMENTED** [review]
Docstrings look good 👍 📜 tests still pass

**Review by ben-d-marks — COMMENTED** [review]
Got it to work with these changes 👍

**Comment by juliakimchung** [comment]
👍 Docstrings look awesome, Good job(Danny and Sam) figuring out the solution for passing the test!!

**Review by samphillips1879 — COMMENTED** [review]
good job on that P.E.P. 8 🚀

**Pull request merged**